src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1962,6 +1974,13 @@ emit_vector_2_3_4 (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *f
 	}
 	case SN_get_Zero:
 		return emit_xzero (cfg, klass);
+	case SN_get_UnitX:
+	case SN_get_UnitY:
+	case SN_get_UnitZ:
+	case SN_get_UnitW: {
+		// FIXME:
+		return NULL;
```

Comment:
Mono doesn't have an easy way to create SIMD constants that I could find.

These are simply:
* `UnitX: 1, 0, 0, 0`
* `UnitY: 0, 1, 0, 0`
* `UnitZ: 0, 0, 1, 0`
* `UnitW: 0, 0, 0, 1`